### ISSUE TITLE
[Bug] or maybe [feature request] - mounted turrets cannot target entities, see your own stats

### ISSUE THREAD

**Issue by nokturnihs** [tsunamayo/Starship-EVO#2347]
### Short Description of the issue:

So I don't know if this should be a bug or a feature request but using a turret via a camera and yoke while in the turret you cannot target nearby ships or see your ships stats.

### Steps to reproduce the issue:

Create a manned turret with a mounted camera.

**Comment by tsunamayo** [comment]
Yes this a bit halfway ;) It is an expected feature that I didnt worked on yet. Thanks!
